cmake/windows-setup.cmake

```diff
@@ -92,7 +92,9 @@ if(FLB_WINDOWS_DEFAULTS)
   set(FLB_FILTER_RECORD_MODIFIER Yes)
   set(FLB_FILTER_REWRITE_TAG    Yes)
   set(FLB_FILTER_GEOIP2         Yes)
-  set(FLB_FILTER_AWS         Yes)
+  set(FLB_FILTER_AWS            Yes)
+  set(FLB_FILTER_ECS            Yes)
+  set(FLB_FILTER_WASM           No)
```

Comment:
Could you remove this line from this 1.9 targeted PR?
fluent-bit 1.9 does not bundle wasm-micro-runtime and does not have wasm support.

Reply:
Oops, this came from merge conflict. Will fix.